.github/workflows/ci.yml

```diff
@@ -60,7 +60,7 @@ jobs:
         run: deno upgrade --canary
 
       - name: Format
-        run: deno fmt --check
+        run: deno fmt --check --ignore=node/_module/node_module
 
       - name: Lint
-        run: deno lint --unstable
+        run: deno lint --unstable --ignore=node/_module/node_modules
```

Comment:
Can you add ignores inline to appropriate files so we can keep these commands without additional arguments?